### DISCUSSION TITLE
how to Listen RealTime  supabse.rpc("url"),

### DISCUSSION THREAD

**Discussion by amresh4448** [supabase/supabase#18348] [General]
final res = await supabase.rpc("get_stores").asStream();
i used like that but realtime is on but there is no changes is UI in flutter  please give me solution

**Comment by realitymolder** [comment]
> ```
>   final res = await supabase.rpc("get_stores").asStream();
> ```
> 
> i used like that but realtime is on but there is no changes is UI in flutter please give me solution

hey there, good question. did you figure it out?
